### DISCUSSION TITLE
o-say-can-you-see/

### DISCUSSION THREAD

**Discussion by giscus** [ironicbadger/blog.ktz.me#1] [Announcements]
# o-say-can-you-see/

And so begins a new adventure, this time a big one. The biggest of 'em all... The USA.  Starting tomorrow, the next 3 weeks ought to be pretty special. We begin by flying to LA, driving up the Pacific Coast highway to San Francisco and Napa followed by a tour

https://blog.ktz.me/o-say-can-you-see/

**Comment by ironicbadger** [comment]
Test.
